src/types/index.ts

```diff
@@ -95,12 +102,18 @@ export type AnonymizedQueryResult = {
   summary: AnonymizationSummary;
 };
 
-export type AnonymizedResultColumn = {
-  name: string;
-  type: AnonymizedColumnType;
-};
-
-export type AnonymizedColumnType = ColumnType | 'aggregate';
+export type AnonymizedResultColumn =
+  | {
+      name: string;
+      type: TooltipColumnType;
+      valueToTooltip: TooltipFunction;
```

Comment:
maybe `tooltipRenderer` or `tooltipGenerator`?

Reply:
True, nothing is being rendered yet, but something is being created, made or formatted.

Reply:
I think it's down to `tooltipCallback` or `tooltipMaker` then. 

(`formatter` is too weak, since there's logic and data (`binSize`) involved.).

Actually... the difficulty in picking a name here convinces me this smells of bad abstraction. I'll spend a second to come up with an alternative to this callback model.

Reply:
see https://github.com/diffix/desktop/pull/255